packages/file/src/download.coffee.md

```diff
@@ -201,15 +201,15 @@ It would be nice to support alternatives sources such as FTP(S) or SFTP.
       # If we know the source signature and if the target file exists
       # we compare it with the target file signature and stop if they match
       if typeof source_hash is 'string'
-        {status} = await @call shy: true, ->
+        {match} = await @call shy: true, ->
           log message: "Shortcircuit check if provided hash match target", level: 'WARN', module: 'nikita/lib/file/download'
           try
             {hash} = await @fs.hash config.target, algo: algo
-            source_hash is hash
+            match: source_hash is hash
           catch err
             throw err if err.code isnt 'NIKITA_FS_STAT_TARGET_ENOENT'
-            true
-        return true unless status
+            match: true
+        return true unless match
```

Comment:
Using status was fine, even if match is now more explicit. However, the overall usage of match, and status before, is still confusing to read. Mayber there is no possibility to simplify it, I'll let you judge it.